reana_server/cli.py

```diff
@@ -51,6 +51,19 @@
 .. code-block :: bash
 
     $ flask users get --admin-access-token=<token>
+
+To export, all users in current REANA cluster:
+
+.. code-block :: bash
+
+    # users will be exported in CSV format
+    $ flask export --admin-access-token=<token>
```

Comment:
What do you think about this?
```suggestion
    $ flask export --admin-access-token=<token> > users.csv
```
Then we can save having the comment just before.